Sources/SwiftConcurrentSequence/Array+ConcurrentMap.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Created by Dan Federman on 4/30/22.
+//  Copyright © 2022 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS"BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Dispatch
+
+extension Array {
```

Comment:
There's also an extension on `Sequence` in this commit that utilizes the methods defined here. I defined methods directly on `Array` to prevent the need to create a new array (via `[Element](self)`) when the receiver is _already_ an array.